android/build.gradle

```diff
@@ -6,16 +6,20 @@ ext {
 }
 
 buildscript {
+    ext.kotlin_version = "1.6.21"
+
     repositories {
         google()
         jcenter()
     }
     dependencies {
         classpath 'com.android.tools.build:gradle:4.2.1'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:${kotlin_version}"
     }
 }
 
 apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
```

Comment:
There is a pattern in which there should be a project-based build.gradle file and a specific, library related one.
Here it's all-in-one. IMO there should be `android/build.gradle` and then `android/library/build.gradle` with library-specific configuration, which will consist of `android` block and `apply plugin`s

Reply:
What the point of separating them in the context of wrappers? We're not going to add any complex config or additional dependencies there

Reply:
E.g. single build.gradle for all the wrappers

Reply:
Hm, maybe that makes sense. But I need to check if we really have the same config in all platforms. I would postpone it to the next stage (written down to backlog)